.github/sync.yml

```
@@ -1,5 +1,10 @@
 group:
   # All repositories that need to sync common files
+  - files:
+      - source: .github/GitReleaseManager/.templates
+        dest: .templates
+    repos: |
+      chocolatey/choco

```

We should probably sync this to `chocolatey/repository-template`.
Unsure if that repository have the proper access for the `choco-bot` though.

---

That is definitely a good idea, yes.  Would you prefer to do that first, or start with chocolatey/choco?

---

I don't mind either way TBH.

But if `choco-bot` already have permissions on `repository-template`, let us just do both repositories at the same time.

```suggestion
      chocolatey/choco
      chocolatey/repository-template
```